src/theme/components/link.ts

```diff
@@ -45,6 +60,43 @@ const variants = {
       textDecoration: "none",
     },
   },
+  buttonPrimary: {
```

Comment:
I forgot that we already had a number of reusable style objects in place for `Link`.  Love it.